docs/src/main/asciidoc/health-guide.adoc

```diff
@@ -56,141 +58,171 @@ which is an implementation of the MicroProfile Health specification used in Quar
 
 == Running the health check
 
-Importing the `smallrye-health` extension directly exposes a single REST endpoint at 
-the `/health` endpoint that can be used to run the health check procedures:
+Importing the `smallrye-health` extension directly exposes three REST endpoints:
+
+- `/health/live` - The application is up and running.
+- `/health/ready` - The application is ready to serve requests.
+- `/health` - Accumulating all health check procedures in the application.
```

Comment:
Someone told me it was deprecated? Should we talk about it?

Reply:
The qualifier is, not the endpoint, at least not in 2.0 up to my knowledge.